src/pymor/models/iosys.py

```diff
@@ -70,6 +71,21 @@ class LTIModel(Model):
     sampling_time
         `0` if the system is continuous-time, otherwise a positive number that denotes the
         sampling time (in seconds).
+    T
+        The final time T.
+    initial_data
+        The initial data `x_0`. Either a |VectorArray| of length 1 or
```

Comment:
```suggestion
        The initial data :math:`x_0`. Either a |VectorArray| of length 1 or
```

Maybe use mathematical notation since `x_0` does not refer to a variable anyway.

Reply:
Also, is initial data an established term? I never heard of it (not that that means anything :D)

Reply:
> Maybe use mathematical notation since `x_0` does not refer to a variable anyway.

This was copied from `InstationaryModel`. I remember that at some point we said that we should reduce the amount of `:math:` to make documentation more readable in the terminal, but that was a discussion among people who rarely use HTML documentation.

> Also, is initial data an established term? I never heard of it (not that that means anything :D)

That was also copied from `InstationaryModel`, so @sdrave would be a better person to ask.

Reply:
convinced